ranger-plugin/admin-plugin/src/main/java/org/apache/hawq/ranger/service/HawqClient.java

```diff
@@ -74,7 +72,7 @@
     private static final String DEFAULT_DATABASE = "postgres";
     private static final String DEFAULT_DATABASE_TEMPLATE = "DBTOBEREPLACEDINJDBCURL";
     private static final String JDBC_DRIVER_CLASS = "org.postgresql.Driver";
-
+    private static final String jaasApplicationName = "pgjdbc";
```

Comment:
Suggest use all capital letters for static final variable, follow the name convention above, like XXX_XXX_XXX.